glib/Gir.toml

```diff
@@ -51,17 +51,38 @@ manual = [
 name = "GLib.*"
 status = "generate"
     [[object.function]]
-    name = "clear_error"
-    #unusable
-    ignore = true
+    pattern = "(assert_warning|assertion_message|assertion_message_cmpnum|assertion_message_cmpstr|warn_message|return_if_fail_warning)"
+    ignore = true # Not useful assertions functions
     [[object.function]]
-    name = "propagate_error"
-    #empty first parameter
-    ignore = true
+    pattern = "(set_print_handler|set_printerr_handler|vasprintf|vfprintf|vprintf|vsnprintf|vsprintf|stpcpy|sprintf|snprintf|printf|printf_string_upper_bound|printerr|prefix_error|print|fprintf)"
+    ignore = true # Not useful string functions
     [[object.function]]
-    name = "set_error_literal"
-    #empty first parameter
-    ignore = true
+    pattern = "(malloc|malloc0|malloc0_n|malloc_n|memdup|memdup2|realloc|realloc_n|try_malloc||try_malloc0|try_malloc0_n|try_malloc_n|try_realloc|try_realloc_n|free)"
+    ignore = true # Not useful allocations functions
+    [[object.function]]
+    pattern = "(slice_get_config|slice_get_config_state|slice_set_config|slice_alloc|slice_alloc0|slice_copy|slice_free1|slice_free_chain_with_offset)"
+    ignore = true # Not useful slices functions
+    [[object.function]]
+    pattern = "(path_is_absolute|build_filename|build_filename_valist|build_filenamev|build_path|build_pathv|canonicalize_filename|chdir)"
```

Comment:
You might also want to ignore then:
  - `g_path_.*`
  - `g_basename`
  - `g_canonicalize_filename`